### PR TITLE
feat: make running locally simpler and faster

### PR DESCRIPTION
**Pull request by norbinsh**
Addresses #2 

- moved hard coded strings to mandatory environment variables
- updated readme a bit
- added optional running against a local mongodb container
- added requirements.txt

Let me know if you don't like any of the changes and I can make adjusments.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
 from commands.birthdays.birthdays_handler import handle_birthdays
 from commands.events.events_handler import handle_events
 from commands.help import help_command
 
+TELEGRAM_TOKEN = os.environ['TELEGRAM_TOKEN']
 
 def main():
-    updater = Updater(token='YOUR_TOKEN', use_context=True)
+    updater = Updater(token=TELEGRAM_TOKEN, use_context=True)
+    print("YOUR SOCIAL LIFE BOT started")
```
Comment (DanTcheche) [inline main.py]: Thank you very much for your collaboration. Please remove this print. I will merge this as soon as it's done.

Comment (norbinsh) [reply main.py]: done!

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+APScheduler==3.6.3
+cachetools==4.2.2
+certifi==2021.10.8
+dnspython==1.16.0
+pymongo==3.12.0
+python-telegram-bot==13.7
+pytz==2021.3
+six==1.16.0
+tornado==6.1
```
Comment (DanTcheche) [inline requirements.txt:9]: Why are tornado, certifi, six, cachetools and APScheduler in this requirements?

Comment (norbinsh) [reply requirements.txt:9]: they are dependencies that pip installs when you install pymongo and python-telegram-bot, i tested it on a fresh virtual environment. do you want me to remove and only leave direct dependencies?

**Pull request merged**